src/Lykke.Service.PayInternal/Controllers/PaymentRequestsController.cs

```diff
@@ -245,33 +224,31 @@ public class PaymentRequestsController : Controller
 
                 return Ok(Mapper.Map<RefundResponseModel>(refundResult));
             }
-            catch (RefundOperationFailedException refundFailedEx)
+            catch (RefundOperationFailedException e)
             {
-                await _log.WriteErrorAsync(nameof(RefundAsync), new {errors = refundFailedEx.TransferErrors}.ToJson(),
-                    refundFailedEx);
+                _log.Error(e, new {errors = e.TransferErrors});
             }
-            catch (AssetUnknownException assetEx)
+            catch (AssetUnknownException e)
             {
-                await _log.WriteErrorAsync(nameof(RefundAsync), new {assetEx.Asset}.ToJson(), assetEx);
+                _log.Error(e, new {e.Asset});
             }
-            catch (AssetNetworkNotDefinedException networkEx)
+            catch (AssetNetworkNotDefinedException e)
             {
-                await _log.WriteErrorAsync(nameof(RefundAsync), new {networkEx.AssetId}.ToJson(), networkEx);
+                _log.Error(e, new {e.AssetId});
             }
-            catch (Exception ex)
+            catch (Exception e)
             {
-                await _log.WriteErrorAsync(nameof(RefundAsync), request.ToJson(), ex);
+                _log.Error(e, request);
```

Comment:
Alexey Novicikhin usually suggests to save in context only data wich is used for searching by fields in kibana.